tools/wurf_options.py

```diff
@@ -63,6 +66,20 @@ def resolve(ctx):
 
 class WurfOptions(Options.OptionsContext):
 
+    def execute(self):
+
+        # Execute the context, this will go out and invoke the option(...)
+        # function defined in the wscript. We have some extra tools that
+        # also define the option(...) function to avoid having to call load
+        # in all wscripts to initialize those, we can do it here.
+
+        self.load('wurf_common_tools')
```

Comment:
Furthermore, can we assume that wurf_common_tools will be loaded automagically in the configure and build steps? (it has to be loaded before anything else is executed in those functions)